Code/PgSQL/rdkit/mol_op.c

```diff
@@ -182,9 +182,9 @@ mol_substruct_count(PG_FUNCTION_ARGS) {
   PG_RETURN_##ret( func(i) );                                           \
 }
   
-MOLDESCR(amw,MolAMW,FLOAT4)
-MOLDESCR(logp,MolLogP,FLOAT4)
-MOLDESCR(tpsa,MolTPSA,FLOAT4)
+MOLDESCR(amw,(float)MolAMW,FLOAT4)
```

Comment:
I'm sure I'm missing something obvious, but it's not clear to me why all of these casts to float are required.